images.yaml

```diff
@@ -24,3 +24,10 @@ Aegir:
     ports:
         8888: auto
         8787: auto
+Rapidsai:
```

Comment:
No need to check this into version control. Just modify the `yaml` directly on the box. If you check the `.gitignore` file, you'll see a line to explicitly ignore changes to `images.yaml`.

Reply:
Ah, I saw the entries for `aegir` and `chill` and thought it would make sense to add new projects to version control. How are we deciding which projects should be tracked by version control in `images.yaml`?

Reply:
That's a very good point. I suppose that short-lived images should not be version-controlled. I put the `aegir` and `chill` in there just to show the template, but it really doesn't make sense for them to stay in the file since those images will eventually be deleted when their respective projects end. I never actually version-controlled those images in [ElderResearch/gpu_docker](https://github.com/ElderResearch/gpu_docker).
Perhaps that is how we decide which images to include in `images.yaml`; whether the images' docker files are version-controlled in the sister repo.